src/core/mac/mac.cpp

```diff
@@ -748,7 +768,9 @@ void Mac::HandleBeginTransmit(void)
 
 extern "C" void otPlatRadioTransmitDone(otInstance *aInstance, bool aRxPending, ThreadError aError)
 {
+    otLogFuncEntryMsg("%!otError!, aRxPending=%u", aError, aRxPending ? 1 : 0);
```

Comment:
BTW, if you are interested, this is the little bit of magic that makes enum to string work for WPP logging (`platform/logging-windows.h`):

```
// begin_wpp config
// CUSTOM_TYPE(otError, ItemEnum(ThreadError));
// CUSTOM_TYPE(otDeviceRole, ItemEnum(otDeviceRole));
// end_wpp
```